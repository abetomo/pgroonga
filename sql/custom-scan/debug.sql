/* Temporary test to be used during implementation of a custom scan. */

CREATE TABLE memos (content text);
CREATE INDEX memos_content ON memos USING pgroonga (content);
INSERT INTO memos VALUES ('PGroonga');
INSERT INTO memos VALUES ('Groonga');
INSERT INTO memos VALUES ('Mroonga');

SET pgroonga.enable_custom_scan = on;

EXPLAIN (COSTS OFF) SELECT * FROM memos where content &@ 'Groonga';
SELECT * FROM memos where content &@ 'Groonga';

DROP TABLE memos;
