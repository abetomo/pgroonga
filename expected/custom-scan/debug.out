/* Temporary test to be used during implementation of a custom scan. */
CREATE TABLE memos (content text);
CREATE INDEX memos_content ON memos USING pgroonga (content);
INSERT INTO memos VALUES ('PGroonga');
SET pgroonga.enable_custom_scan = on;
EXPLAIN (COSTS OFF) SELECT * FROM memos;
             QUERY PLAN              
-------------------------------------
 Custom Scan (PGroongaScan) on memos
   PGroongaScan: DEBUG
(2 rows)

SELECT * FROM memos;
 content 
---------
 Groonga
(1 row)

DROP TABLE memos;
